### PR TITLE
Add some arabic translation

### PR DESCRIPTION
**Pull request by Mabrouk12300**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Mabrouk12300

## Language: 

- Arabic

## Terms defined:

- aggregation
- Aggregation function
- Algorithm
- API
- asynchronous
- Big Data

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -463,8 +463,12 @@
     term: "agregação"
     def: >
       Combinar vários valores em um, por exemplo, somando um conjunto numérico ou
-      concatenando um conjunto de caracteres.
-
+   concatenando um conjunto de caracteres.
+  ar:
+    term: "تجميع"
+    def: >
+   دمج مجموعة من القيم في قيمة واحدة، مثال، حمع مجموعة من الأرقام
```
Comment (BatoolMM) [inline glossary.yml:470]:
```suggestion
      دمج مجموعة من القيم في قيمة واحدة، مثال، حمع مجموعة من الأرقام
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]
ترجمة جدا ممتازة، الشئ الوحيد هو ان هذا 
YAML file 
لذلك الفراغ قبل بداية التريف مهم 
أيضا الروابط  والوصلات قد لاتعمل لذلك راح نتأكد لما ندمج البي آر لاحقا 

عذرا جدا على التأخر

File: glossary.yml [hunk 2]
```diff
@@ -493,13 +497,19 @@
     def: >
       'n Funksie wat 'n groep waardes saamvoeg, bv. `sum` (summering) en `max`
       (maksimale waarde)
+  ar:
+  term: "دالة التجميع"
+  def: >
+  دالة تدمج العديد من القيم في قيمة واحدة ، مثل
```
Comment (BatoolMM) [inline glossary.yml:503]:
```suggestion
     دالة تدمج العديد من القيم في قيمة واحدة ، مثل
```

File: glossary.yml [hunk 3]
```diff
@@ -493,13 +497,19 @@
...
+  [جمع](#sum)
```
Comment (BatoolMM) [inline glossary.yml:504]:
```suggestion
     [جمع](#sum)
```

File: glossary.yml [hunk 4]
```diff
@@ -493,13 +497,19 @@
...
+  أو
```
Comment (BatoolMM) [inline glossary.yml:505]:
```suggestion
     أو
```

File: glossary.yml [hunk 5]
```diff
@@ -493,13 +497,19 @@
...
+  [أقصى](#max).
```
Comment (BatoolMM) [inline glossary.yml:506]:
```suggestion
     [أقصى](#max).
```

File: glossary.yml [hunk 6]
```diff
@@ -568,6 +578,11 @@
     def: >
       Un algorithme est un ensemble d'étapes ou de règles à suivre pour réaliser une certaine tâche.
       En informatique, un algorithme est un ensemble d'instructions présentes dans un programme qui permet de résoudre un problème informatique.
+   ar:
+   term: "خوارزمية"
+   def: >
+   الخوارزمية عبارة عن مجموعة من الخطوات والمقدمات أوالقواعد التي يجب اتباعها لتحقيق مهمة معين.
```
Comment (BatoolMM) [inline glossary.yml:584]:
```suggestion
      الخوارزمية عبارة عن مجموعة من الخطوات والمقدمات أوالقواعد التي يجب اتباعها لتحقيق مهمة معين.
```

File: glossary.yml [hunk 7]
```diff
@@ -568,6 +578,11 @@
...
+   في علوم الحاسب، الخوارزمية هي مجموعة من التعليمات في برنامج كمبيوتر تحل مشكلة حاسوبية.
```
Comment (BatoolMM) [inline glossary.yml:585]:
```suggestion
      في علوم الحاسب، الخوارزمية هي مجموعة من التعليمات في برنامج كمبيوتر تحل مشكلة حاسوبية.
```

File: glossary.yml [hunk 8]
```diff
@@ -745,8 +759,12 @@
     def: >
       Un conjunto de funciones y procedimientos proporcionados por una libreria de software o servicio web
       atraves del cual otra aplicación se puede comunicar. Una API no es el código,
-      la base de datos o el servidor; es el punto de acceso.
-
+      la base de datos o el servidor; es el punto de acceso.    
+  ar:
+    term: "واجهة التطبيق البرمجية"
+    def: >
+   مجموعة من الوظائف والإجراءات التي توفرها مكتبات البرامج أو خدمة الويب والتي يمكن من خلالها  لتطبيق آخر التواصل معها.
```
Comment (BatoolMM) [inline glossary.yml:766]:
```suggestion
      مجموعة من الوظائف والإجراءات التي توفرها مكتبات البرامج أو خدمة الويب والتي يمكن من خلالها  لتطبيق آخر التواصل معها.
```

File: glossary.yml [hunk 9]
```diff
@@ -745,8 +759,12 @@
...
+   واجهة التطبيق البرمجية ليست الكود أو قاعدة البيانات أو الخادم: إنها نقطة الوصول.
```
Comment (BatoolMM) [inline glossary.yml:767]:
```suggestion
      واجهة التطبيق البرمجية ليست الكود أو قاعدة البيانات أو الخادم: إنها نقطة الوصول.
```

File: glossary.yml [hunk 10]
```diff
@@ -875,6 +894,12 @@
     term: "ASCII"
     def: >
       'n Standaard manier om karakters wat algemeen in Wes-Europese tale gebruik word,
+      voor te stel as 7- of 8-bis heelgetalle. ASCII word vervang deur [Unicode](#unicode).
+  ar:
+    term: "أسكي"
```
Comment (BatoolMM) [inline glossary.yml:899]:
```suggestion
    term: "آسكي"
```

File: glossary.yml [hunk 11]
```diff
@@ -875,6 +894,12 @@
...
+    def: >
+  طريقة قياسية لتمثيل الأحرف المستخدمة بشكل شائع في لغات أوروبا الغربية كأعداد صحيحة من 7 أو 8 بت،
```
Comment (BatoolMM) [inline glossary.yml:901]:
```suggestion
     طريقة قياسية لتمثيل الأحرف المستخدمة بشكل شائع في لغات أوروبا الغربية كأعداد صحيحة من ٧ أو ٨ بت،
```

File: glossary.yml [hunk 12]
```diff
@@ -875,6 +894,12 @@
...
+   يحل محلها الآن   [الترميز الموحد] (#unicode).
```
Comment (BatoolMM) [inline glossary.yml:902]:
```suggestion
      يحل محلها الآن   [الترميز الموحد] (#unicode).
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
Thank you for your contribution @Mabrouk12300 , just a few comments. Let me know if anything isn't clear!

File: glossary.yml [hunk 13]
```diff
@@ -463,8 +463,12 @@
     term: "agregação"
     def: >
       Combinar vários valores em um, por exemplo, somando um conjunto numérico ou
-      concatenando um conjunto de caracteres.
-
+   concatenando um conjunto de caracteres.
```
Comment (BatoolMM) [inline glossary.yml:466]:
```suggestion
      concatenando um conjunto de caracteres.
```

File: glossary.yml [hunk 14]
```diff
@@ -745,8 +759,12 @@
     def: >
       Un conjunto de funciones y procedimientos proporcionados por una libreria de software o servicio web
       atraves del cual otra aplicación se puede comunicar. Una API no es el código,
-      la base de datos o el servidor; es el punto de acceso.
-
+      la base de datos o el servidor; es el punto de acceso.
```
Comment (BatoolMM) [inline glossary.yml:762]:
```suggestion
      la base de datos o el servidor; es el punto de acceso.    
```

File: glossary.yml [hunk 15]
```diff
@@ -875,6 +894,12 @@
     term: "ASCII"
     def: >
       'n Standaard manier om karakters wat algemeen in Wes-Europese tale gebruik word,
+      voor te stel as 7- of 8-bis heelgetalle. ASCII word vervang deur [Unicode](#unicode).
```
Comment (BatoolMM) [inline glossary.yml:897]:
```suggestion
      voor te stel as 7- of 8-bis heelgetalle. ASCII word vervang deur [Unicode](#unicode).
```

File: glossary.yml [hunk 16]
```diff
@@ -975,9 +999,15 @@
   af:
     term: "asinchronies"
     def: >
-      "Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer
-      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n
-      bewerking wat onafhanklik van 'n ander een plaasvind."
+      Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer
```
Comment (BatoolMM) [inline glossary.yml:1002]:
```suggestion
      Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer 
```

File: glossary.yml [hunk 17]
```diff
@@ -975,9 +999,15 @@
   af:
     term: "asinchronies"
     def: >
-      "Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer
-      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n
-      bewerking wat onafhanklik van 'n ander een plaasvind."
+      Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer 
+      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n
```
Comment (BatoolMM) [inline glossary.yml:1003]:
```suggestion
      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n 
```

File: glossary.yml [hunk 18]
```diff
@@ -975,9 +999,15 @@
   af:
     term: "asinchronies"
     def: >
-      "Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer
-      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n
-      bewerking wat onafhanklik van 'n ander een plaasvind."
+      Asinchronies is die teenoorgestelde van sinchronies wat beteken dat twee of meer 
+      aktiwiteite tergelykertyd plaasvind. In programering is 'n asinchorniese bewerking 'n 
+      bewerking wat onafhanklik van 'n ander een plaasvind.
```
Comment (BatoolMM) [inline glossary.yml:1004]:
```suggestion
      bewerking wat onafhanklik van 'n ander een plaasvind.
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
It seems that the indentation of some of the text was changed when you added the Arabic translation, can you commit the changes I suggested and see if it fixes the CI?

**Comment by Mabrouk12300** [comment]
hi
Sure
I will do it now

[image: photo]
*Dr. Mabrouk boutagouga د. مبروك بوطقوقة *
أستاذ محاضر، جامعة باتنة 1 | المشرف العام لمركز فاعلون, رئيس تحرير مجلة
أنثروبولوجيا | مؤسس موقع أرنتروبوس

Whatsapp:(00213)5 <(00213)0669689110>50643288 | ***@***.***
***@***.***>

http://www.aranthropos.com | http://www. <http://www.fa3iloon.org>
fa3iloon.org <http://www.fa3iloon.org>
كلية اللغة والأدب العربي والفنون، جامعة باتنة 1، طريق بسكرة الجزائر 5000
<http://www.facebook.com/mabroukboutagouga> <http://Twitter.com/boutagouga>
<http://www.linkedin.com/in/mabrouk-boutagouga-44498a6a>


Le jeu. 2 déc. 2021 à 16:16, Batool Almarzouq ***@***.***> a
écrit :

File: glossary.yml [hunk 19]
```diff
@@ -501,13 +507,19 @@
     def: >
       'n Funksie wat 'n groep waardes saamvoeg, bv. `sum` (summering) en `max`
       (maksimale waarde)
+  ar:
+  term: "دالة التجميع"
```
Comment (BatoolMM) [inline glossary.yml:511]:
```suggestion
    term: "دالة التجميع"
```

File: glossary.yml [hunk 20]
```diff
@@ -501,13 +507,19 @@
...
+  def: >
```
Comment (BatoolMM) [inline glossary.yml:512]:
```suggestion
    def: >
```

File: glossary.yml [hunk 21]
```diff
@@ -576,6 +588,11 @@
     def: >
       Un algorithme est un ensemble d'étapes ou de règles à suivre pour réaliser une certaine tâche.
       En informatique, un algorithme est un ensemble d'instructions présentes dans un programme qui permet de résoudre un problème informatique.
+   ar:
+   term: "خوارزمية"
```
Comment (BatoolMM) [inline glossary.yml:592]:
```suggestion
     term: "خوارزمية"
```

File: glossary.yml [hunk 22]
```diff
@@ -576,6 +588,11 @@
...
+   def: >
```
Comment (BatoolMM) [inline glossary.yml:593]:
```suggestion
     def: >
```

**Review by BatoolMM — COMMENTED** [review]
I'm trying to review and fix for indentation

File: glossary.yml [hunk 23]
```diff
@@ -467,13 +467,19 @@
     def: >
       Combinar vários valores em um, por exemplo, somando um conjunto numérico ou
       concatenando um conjunto de caracteres.
+
+  ar:
+    term: "تجميع"
+    def: >
+      دمج مجموعة من القيم في قيمة واحدة، مثال، حمع مجموعة من الأرقام
+   أو تحميع مجموعة من السلاسل.
```
Comment (BatoolMM) [inline glossary.yml:475]:
```suggestion
     أو تحميع مجموعة من السلاسل. 
```

File: glossary.yml [hunk 24]
```diff
@@ -501,13 +507,19 @@
     def: >
       'n Funksie wat 'n groep waardes saamvoeg, bv. `sum` (summering) en `max`
       (maksimale waarde)
+  ar:
+    term: "دالة التجميع"
+    def: >
+     دالة تدمج العديد من القيم في قيمة واحدة ، مثل
```
Comment (BatoolMM) [inline glossary.yml:513]:
```suggestion
       دالة تدمج العديد من القيم في قيمة واحدة ، مثل
```

File: glossary.yml [hunk 25]
```diff
@@ -501,13 +507,19 @@
...
+     [جمع](#sum)
```
Comment (BatoolMM) [inline glossary.yml:514]:
```suggestion
       [جمع](#sum)
```

File: glossary.yml [hunk 26]
```diff
@@ -501,13 +507,19 @@
...
+     أو
```
Comment (BatoolMM) [inline glossary.yml:515]:
```suggestion
       أو
```

File: glossary.yml [hunk 27]
```diff
@@ -501,13 +507,19 @@
...
+     [أقصى](#max).
```
Comment (BatoolMM) [inline glossary.yml:516]:
```suggestion
       [أقصى](#max).
```

File: glossary.yml [hunk 28]
```diff
@@ -576,6 +588,11 @@
     def: >
       Un algorithme est un ensemble d'étapes ou de règles à suivre pour réaliser une certaine tâche.
       En informatique, un algorithme est un ensemble d'instructions présentes dans un programme qui permet de résoudre un problème informatique.
+   ar:
+     term: "خوارزمية"
+   <span class="x x-first x-last">  </span>def: &gt;
```
Comment (BatoolMM) [inline glossary.yml:593]:
```suggestion
    def: >
```

File: glossary.yml [hunk 29]
```diff
@@ -768,6 +789,7 @@
     def: >
       በአንድ የሶፍትዌር ቤተ-መጽሐፍት ወይም ድር የሚሰጡ የተግባሮች እና የአሠራር ሂደቶች ስብስብ ሌላ መተግበሪያ ከእሱ ጋር መገናኘት የሚችልበት አገልግሎት። ኤፒ አይ ኮድ አይደለም  ፣ የመረጃ ቋቱ ወይም አገልጋዩ የመዳረሻ ነጥብ ነው ፡፡
 
+
```
Comment (BatoolMM) [inline glossary.yml:792]:
```suggestion
```

File: glossary.yml [hunk 30]
```diff
@@ -880,6 +902,12 @@
     term: "ASCII"
     def: >
       'n Standaard manier om karakters wat algemeen in Wes-Europese tale gebruik word,
+      voor te stel as 7- of 8-bis heelgetalle. ASCII word vervang deur [Unicode](#unicode).
+  ar:
+    term: "آسكي"
+    def: >
+     طريقة قياسية لتمثيل الأحرف المستخدمة بشكل شائع في لغات أوروبا الغربية كأعداد صحيحة من ٧ أو ٨ بت،
```
Comment (BatoolMM) [inline glossary.yml:909]:
```suggestion
       طريقة قياسية لتمثيل الأحرف المستخدمة بشكل شائع في لغات أوروبا الغربية كأعداد صحيحة من ٧ أو ٨ بت،
```

File: glossary.yml [hunk 31]
```diff
@@ -880,6 +902,12 @@
...
+      يحل محلها الآن   [الترميز الموحد] (#unicode).
```
Comment (BatoolMM) [inline glossary.yml:910]:
```suggestion
       يحل محلها الآن   [الترميز الموحد] (#unicode).
```

File: glossary.yml [hunk 32]
```diff
@@ -467,13 +467,19 @@
     def: >
       Combinar vários valores em um, por exemplo, somando um conjunto numérico ou
       concatenando um conjunto de caracteres.
+
+  ar:
+    term: "تجميع"
+    def: >
+      دمج مجموعة من القيم في قيمة واحدة، مثال، حمع مجموعة من الأرقام
+     أو تحميع مجموعة من السلاسل.
```
Comment (BatoolMM) [inline glossary.yml:475]:
```suggestion
```

File: glossary.yml [hunk 33]
```diff
@@ -576,6 +588,11 @@
     def: >
       Un algorithme est un ensemble d'étapes ou de règles à suivre pour réaliser une certaine tâche.
       En informatique, un algorithme est un ensemble d'instructions présentes dans un programme qui permet de résoudre un problème informatique.
+   ar:
+     term: "خوارزمية"
+    def: >
```
Comment (BatoolMM) [inline glossary.yml:593]:
```suggestion
   def: >
```

File: glossary.yml [hunk 34]
```diff
@@ -576,6 +587,11 @@
     def: >
       Un algorithme est un ensemble d'étapes ou de règles à suivre pour réaliser une certaine tâche.
       En informatique, un algorithme est un ensemble d'instructions présentes dans un programme qui permet de résoudre un problème informatique.
+  ar:
+     term: "خوارزمية"
+   def: >
```
Comment (BatoolMM) [inline glossary.yml:592]:
```suggestion
    def: >
```

File: glossary.yml [hunk 35]
```diff
@@ -1297,6 +1328,10 @@
     def: >
       Any data that until recently was too big for most people to work with on a
       single computer.
+  ar:
+    term:"البيانات الضخمة"
```
Comment (BatoolMM) [inline glossary.yml:1332]:
```suggestion
    term: "البيانات الضخمة"
```

**Comment by BatoolMM** [comment]
@fmichonneau Sorry but would you mind having a look at this! I tried to fix the white spaces/indentation but it's a bit challenging working with RTL in YAML - the CI for the lint still failing!!

**Comment by fmichonneau** [comment]
Thanks @BatoolMM! it looks like the errors come from somewhere else in the file, so I'm going to merge this and fix the errors reported by the lintr.

**Comment by BatoolMM** [comment]
Thank you @fmichonneau!

**Comment by elletjies** [comment]
@all-contributors please add @Mabrouk12300 for translation

**Comment by elletjies** [comment]
@all-contributors please add @BatoolMM for review, translation

**Pull request merged**